homeassistant/components/uvc/camera.py

```diff
@@ -106,6 +107,20 @@ def motion_detection_enabled(self):
         caminfo = self._nvr.get_camera(self._uuid)
         return caminfo['recordingSettings']['motionRecordEnabled']
 
+    @property
+    def supported_features(self):
+        """Return supported features."""
+        return SUPPORT_STREAM
+
+    @property
+    def stream_source(self):
+        """Return the stream source."""
+        caminfo = self._nvr.get_camera(self._uuid)
+        if caminfo['channels'][0]['isRtspEnabled']:
+            stream = caminfo['channels'][0]['rtspUris'][0].split(':')[2]
+            return 'rtsp://%(host)s:%(stream)s' % dict(
+                host=self._nvr._host, stream=stream)
```

Comment:
You can either disable lint on this line, or pass config address through to this class. Either will work.

Reply:
how would I go about that? `def stream_source(self, nvr=config[CONF_NVR])`?

Reply:
could you explain why `._host` is bad if it's already doing things like `self._nvr.get_camer()`?

Reply:
Variable names starting with an underscore indicate that it's a private variable. So accessing your own private variables, like `self._nvr` is fine. Accessing other ones is not fine, as there is no guarantee that they stay the same.

Reply:
I think I did it right. Could you double check for me? 🙏 

Thanks!